manifests/infra/grafana-ingress/overlays/development/ingress.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  name: grafana
+  namespace: monitoring
+spec:
+  virtualhost:
+    fqdn: grafana.dev.cloudnativedays.jp
+    tls:
+      secretName: cert-manager/wildcard-dev-cloudnativedays-jp
+  routes:
+  - services:
+      - name: grafana
+        port: 80
```

どちらでもよい細かいことですが、他に合わせるならインデントはこうですかね。
```suggestion
    - name: grafana
      port: 80
```

---

むしろ公式がこっちのインデントだったので合わせた感じですが、特に好みはないのでどちらでもって感じです。